k8s/namespaces/reform-scan/reform-scan-notification-service/prod.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: reform-scan-notification-service
+  annotations:
+    fluxcd.io/automated: "true"
+    repository.fluxcd.io/java: java.image
+    filter.fluxcd.io/java: glob:prod-*
```

Comment:
no needed

Reply:
They exist in other project yaml files. I would like to keep them same

Reply:
https://github.com/hmcts/cnp-flux-config/pull/5364#discussion_r468138377